Classes/PBCommitList.m

```diff
@@ -43,7 +43,10 @@ - (void)keyDown:(NSEvent *)event
 	}
 	else if ([character rangeOfCharacterFromSet:[NSCharacterSet characterSetWithCharactersInString:@"jkcv"]].location == 0)
 		[webController sendKey: character];
-	else
+	else if (([character characterAtIndex:0] == NSDownArrowFunctionKey)
+			 && [event modifierFlags] & NSControlKeyMask) {
+		[controller selectParentCommit:self];
+	} else
```

Comment:
If this doesn't go in the menu bar, it should definitely have some documentation somewhere. (Eventually; doesn't need to block merging.) A list of keyboard shortcuts on the website would be nice, and would be findable from search engines.

Even other menu bar shortcuts would benefit from that too. For myself, I had been selecting and copying SHAs from the HTML diff header until I learned of ⌥⌘C from the recent copy discussions. Mashing modifier keys while browsing through app menus is a good habit, but a big list on the homepage is also nice and obvious.